02-go-level-1/simple/simplenum/simplenum.go

```diff
@@ -0,0 +1,33 @@
+package simplenum
+
+import (
+	"math"
+)
+
+//GetNum is printing simple numbs
```

Comment:
```suggestion
// GetNum is printing simple numbs
```

тут по стандартам пробельчик нужен между `//` и остальным.